src/index.js

```diff
@@ -16,3 +33,70 @@ function toggleMenu() {
 }
 
 hamburgerMenu.addEventListener('click', toggleMenu);
+
+let currentProfile = document.querySelector(
+	'.profile--selected'
+);
+const orgName = document.getElementById('about__name');
+const orgAbout = document.getElementById('about__desc');
+const orgPhoto = document.getElementById('about__photo');
+
+const orgProfiles = document.querySelectorAll('.profile');
+
+// Add event listener to profiles
+for (let i = 0; i < orgProfiles.length; i++) {
+	orgProfiles[i].addEventListener('click', selectProfile);
+	orgProfiles[i].addEventListener('keypress', (e) => {
+		if (handleClick(e)) {
+			selectProfile(e);
+		}
+	});
+}
+
+// Function to select organizer profile -- updates from organizers data object
+function selectProfile(e) {
+	let currentProfileId = e.currentTarget.id;
+
+	if (organizers[currentProfileId]) {
+		// Remove default highlighted profile
+		currentProfile.classList.remove('profile--selected');
+		// Update current profile to be highlighted
+		currentProfile = document.querySelector(
+			`#${currentProfileId}`
+		);
+		currentProfile.classList.add('profile--selected');
+
+		// Update about section content
+		orgName.textContent = organizers[currentProfileId].name;
+		orgAbout.textContent = organizers[currentProfileId].about;
+		orgPhoto.src = organizers[currentProfileId].profilePhoto;
+		orgPhoto.alt = `Kalispell Software Crafters organizer ${organizers[currentProfileId].name}`;
+		}
+		// else {
+	// 	alert('Organizer is not found.');
+	// }
+}
+
+const organizers = {
+	Travis1: {
+		id: 1,
```

Comment:
If I'm understanding right, I don't think the `id` field is used (but maybe we want to swap things around so it is?). Also, if we go with what we have, I think we could probably remove the number from the names (and in the HTML). So we could use `Travis` rather than `Travis1`. 

What do you think? Or did you find that it was needed?

Reply:
I don't think we need the `id` field for the current implementation. Removed them for now and updated the identifying name IDs.

Reply:
Looks good, thanks!